scheduler/scheduler-server/src/test/java/functionaltests/utils/SchedulerTHelper.java

```diff
@@ -752,46 +751,41 @@ public JobInfo waitForEventJobFinished(JobId id) throws Exception {
         return waitForEventJobFinished(getSchedulerInterface(), id);
     }
 
-    /**
-     * @param userInterface
-     * @param id of the job
-     * @return jobInfo, if job went from RUNNNIG or PENDING to FINISHED, otherwise null will be returned
-     * @throws Exception
-     */
     public JobInfo waitForEventJobFinished(Scheduler userInterface, JobId id) throws Exception {
         try {
-            JobEventMonitor monitorRunningToFinished = null;
-            JobEventMonitor monitorPendingToFinished = null;
-            synchronized (getSchedulerMonitorsHandler()) {
-                Optional<JobEventMonitor> opJobMonitor = getSchedulerMonitorsHandler().removeJobEvent(id,
-                                                                                                      SchedulerEvent.JOB_RUNNING_TO_FINISHED);
-                if (opJobMonitor.isPresent()) {
-                    //event occurred, remove it and return associated Job object
-                    return opJobMonitor.get().getJobInfo();
-                }
-                opJobMonitor = getSchedulerMonitorsHandler().removeJobEvent(id, SchedulerEvent.JOB_PENDING_TO_FINISHED);
-                if (opJobMonitor.isPresent()) {
-                    //event occurred, remove it and return associated Job object
-                    return opJobMonitor.get().getJobInfo();
-                }
-
-                monitorRunningToFinished = (JobEventMonitor) getSchedulerMonitorsHandler().getMonitor(new JobEventMonitor(SchedulerEvent.JOB_RUNNING_TO_FINISHED,
-                                                                                                                          id));
-                monitorPendingToFinished = (JobEventMonitor) getSchedulerMonitorsHandler().getMonitor(new JobEventMonitor(SchedulerEvent.JOB_PENDING_TO_FINISHED,
-                                                                                                                          id));
+            return waitForJobEvent(userInterface,
+                                   id,
+                                   700000, // 700s to wait, when overall junit timeout is 800s
```

Comment:
could you reuse CentralPAPropertyRepository.PA_TEST_TIMEOUT.getValue() instead of a hardcoded value ?